WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -114,48 +133,56 @@ private extension ItemListView {
         VStack {
             POSPageHeaderView(items: headerViewItems, trailingContent: {
                 HStack {
-                    if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.searchProductsInPOS),
-                       case .products = selectedItemListType {
-                        searchField.onChange(of: searchTerm) { oldValue, newValue in
-                            selectedItemListType = .products(search: newValue.isNotEmpty)
-
-                            // The debouncing logic is a little tricky, because the loading state is held in the controller.
-                            // Arguably, we should use view state `isSearching` for this, so the UI is independent of the request timing.
-
-                            // As the user types, we don't want to send every keystroke to the remote, so we debounce the requests.
-                            // However, we don't want to debounce the first keystroke of a new search, so that the loading
-                            // state shows immediately and the UI feels responsive.
-
-                            // So, if the last search was finished, we don't debounce the first character. If it didn't
-                            // finish i.e. it is still ongoing, we debounce the next keystrokes by 300ms. In either case,
-                            // the ongoing search is redundant now there's a new search term, so we cancel it.
-                            let shouldDebounceNextSearchRequest = !didFinishSearch
-                            searchTask?.cancel()
-
-                            searchTask = Task {
-                                if shouldDebounceNextSearchRequest {
-                                    try? await Task.sleep(nanoseconds: 300 * NSEC_PER_MSEC)
-                                }
-
-                                guard !Task.isCancelled else { return }
-
-                                didFinishSearch = false
-
-                                await posModel.purchasableItemsSearchController.searchItems(searchTerm: newValue, baseItem: .root)
-
-                                if !Task.isCancelled {
-                                    didFinishSearch = true
+                    if isSearchAllowed {
+                        searchField
```

Comment:
Since we don't have designs, I'm not sure what will and what won't be addressed for this release.

However, I would at least expect a larger search field text.

Reply:
Yeah good point, I'll make it a bit bigger. But for this release, it'll look pretty much as it does here.